src/features/helios/components/garbageCollector/components/GarbageDiscard.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useState } from "react";
+import { useActor } from "@xstate/react";
+
+import { Box } from "components/ui/Box";
+import { Button } from "components/ui/Button";
+
+import { Context } from "features/game/GameProvider";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { Decimal } from "decimal.js-light";
+
+import { DISCARD, DiscardName } from "features/game/types/garbage";
+import { SplitScreenView } from "components/ui/SplitScreenView";
+import { ShopSellDetails } from "components/ui/layouts/ShopSellDetails";
+
+export const GarbageDiscard: React.FC = () => {
+  const [selectedName, setSelectedName] = useState<DiscardName>(DISCARD[0]);
+
+  const { gameService } = useContext(Context);
+  const [
+    {
+      context: { state },
+    },
+  ] = useActor(gameService);
+
+  const inventory = state.inventory;
+
+  const amount = inventory[selectedName] || new Decimal(0);
+
+  const discard = (amount = 1) => {
```

Comment:
Is this default assignment necessary?

Reply:
Modeled on what the Sell tab does.  Would you like to remove the ` = 1` from both of them?